### ISSUE TITLE
https://github.com/HippieStationCode/HippieStation13/pull/3198 Is not ported yet

### ISSUE THREAD

**Issue by Uqualm** [Carbonhell/Hippie-Rebase#102]
You mad men, you must do something about it before it's too late!!!

**Comment by KittyKathy** [comment]
Yeah we need to revoke your git access permanently it's the only solution.

**Comment by QuarianCommando** [comment]
The server's been really nice without anti-stuns being mass produced every round though.

**Comment by Uqualm** [comment]
Tsundere.

**Comment by Sarcalogo** [comment]
Uqualm is me btw.

**Comment by banthebantz** [comment]
Mcbaw said it will be

**Comment by KittyKathy** [comment]
Oh that's actually the PR for anti-stun removal I thought you were still butt hurt and this was the RD armor removal PR. That will be ported when someone actually does it like do it yourself if you want it ported so badly...actually that might be a bad idea you would screw it up we need someone competent to do it like Arc.

**Comment by Sarcalogo** [comment]
KittyKathy, I can handle it, trust me.

**Comment by KittyKathy** [comment]
Haha, nah you don't know what kind of anti-stuns they added and where so I can't get a guarantee that you would remove them all I mean that PR we all forgot to remove Ninja anti-stuns and that Cybernetic implant that reduces stuns.
